admin_ui_support/tests/src/Functional/rest/ConfigSchemaResourceTest.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace Drupal\Tests\admin_ui_support\Functional\rest;
+
+use Drupal\Component\Serialization\Json;
+use Drupal\Core\Url;
+use Drupal\rest\RestResourceConfigInterface;
+use Drupal\Tests\rest\Functional\CookieResourceTestTrait;
+use Drupal\Tests\rest\Functional\ResourceTestBase;
+
+/**
+ * Tests the config schema resource.
+ *
+ * @group user
+ */
+class ConfigSchemaResourceTest extends ResourceTestBase {
+
+  use CookieResourceTestTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $format = 'hal_json';
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $mimeType = 'application/hal+json';
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $auth = 'cookie';
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $resourceConfigId = 'config_schema';
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = ['hal', 'user', 'rest', 'admin_ui_support'];
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setUp() {
+    parent::setUp();
+
+    $auth = isset(static::$auth) ? [static::$auth] : [];
+    $this->provisionResource([static::$format], $auth);
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function provisionResource($formats = [], $authentication = []) {
+    $this->resourceConfigStorage->create([
+      'id' => static::$resourceConfigId,
+      'granularity' => RestResourceConfigInterface::RESOURCE_GRANULARITY,
+      'configuration' => [
+        'methods' => ['GET'],
+        'formats' => $formats,
+        'authentication' => $authentication,
+      ],
+      'status' => TRUE,
+    ])->save();
+    $this->refreshTestStateAfterRestConfigChange();
+  }
+
+  public function testConfigSchema() {
+    $this->initAuthentication();
+    $url = Url::fromRoute('rest.config_schema.GET', ['_format' => static::$format, 'name' => 'system.site']);
+    $request_options = $this->getAuthenticationRequestOptions('GET');
+
+    $response = $this->request('GET', $url, $request_options);
+    $this->assertResourceResponse(200, false, $response, ['config:rest.resource.config_schema', 'config:rest.settings', 'http_response'], [], FALSE, 'MISS');
```

Comment:
I get `Failed asserting that 403 is identical to 200.` here

Reply:
Oh, mh, I actually just c&pasted the test together :(

Reply:
This fixes the test https://github.com/tedbow/drupal-admin-ui/commit/03ea0a8339fd2a27230feb63376562cea6437708

Reply:
Thank you!